### PR TITLE
chore: APP-2680 - Create tests for the most common DAO actions

### PR DESCRIPTION
**Pull request by Barukimang**
## Description

**Flashback:**
- Past months could not advance because of bugs which had to be resolved at package level of Synpress and Cypress
- Talked with the main developer of Synpress and he told me about their new alpha package 'New Dawn': https://hackmd.io/@pmgbeMWzS3q-Bv3v-SNDEw/Sy90jQNZp
- Also he advised me to use Playwright instead of Cypress: less bugs and better suited for Synpress

**Work done in this PR:**
1. Implemented the new alpha package of Synpress, see Docs here: https://synpress.io/
2. Implemented Playwright, see Docs here: https://playwright.dev/
3. Removed folders and files of the 'old Synpress + Cypress' setup
4. Created tests for the most common DAO actions (5 tests)

**To get started:**

1. Ask me for the secrets :)
2. Run `yarn`
3. Start localhost (you might run into an error, in that case delete `"type": "module",` from `package.json`)
4. Run `yarn e2e` (you might run into an error, in that case add `"type": "module",` to `package.json` again, didn't find a proper fix still :) )
5. The Playwright UI should open
6. Run all, or selected tests from the Playwright UI

Task: [APP-2681](https://aragonassociation.atlassian.net/browse/APP-2681)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2681]: https://aragonassociation.atlassian.net/browse/APP-2681?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```
Comment (cgero-eth) [inline .github/workflows/playwright.yml]: Can we update the current e2e-tests workflow instead?

File: package.json [hunk 2]
```diff
@@ -52,6 +53,7 @@
     "graphql-request": "^4.3.0",
     "i18next": "^23.5.1",
     "numeral": "^2.0.6",
+    "playwright-core": "^1.42.1",
```
Comment (cgero-eth) [inline package.json]: Is this dependency required?

File: package.json [hunk 3]
```diff
@@ -15,15 +16,15 @@
...
-    "e2e:reset": "cd tests/e2e && cpx default/runContext.json .",
+    "e2e": "playwright test --ui",
     "sentry:sourcemaps": "sentry-cli sourcemaps inject --org aragonorg --project app-react ./dist && sentry-cli sourcemaps upload --org aragonorg --project app-react ./dist"
   },
   "dependencies": {
     "@apollo/client": "^3.5.8",
     "@aragon/ods": "^1.0.20",
     "@aragon/sdk-client": "^1.22.1",
     "@aragon/sdk-client-common": "^1.14.1",
+    "@playwright/test": "^1.42.1",
```
Comment (cgero-eth) [inline package.json:27]: Can we remove the old cypress dependencies?

File: package.json [hunk 4]
```diff
@@ -6,6 +6,7 @@
   "author": "Aragon Association",
   "homepage": ".",
   "license": "GPL-3.0",
+  "type": "module",
```
Comment (cgero-eth) [inline package.json:9]: We shouldn't add this, let's maybe sync and check how we can make playwright work without type module

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (Barukimang) [reply .github/workflows/playwright.yml]: Deleted the workflow https://github.com/aragon/app/pull/1305/commits/30641aa2d432f9228cc503895eebce4719d653e2
Will update current e2e-tests in later PR

File: package.json (hunk 4 above)
Comment (Barukimang) [reply package.json:9]: Thanks for the fix, build is working fine: https://ipfs.eth.aragon.network/ipfs/Qmc3i8ygSGfTmecqgYkTchmmkB5rhMwTjFatSTpc7gQDwp/

File: package.json (hunk 3 above)
Comment (Barukimang) [reply package.json:27]: I have removed it: https://github.com/aragon/app/pull/1305/commits/669b9815c7a330e6adbf23d6318e6a826a89721e

File: package.json (hunk 2 above)
Comment (Barukimang) [reply package.json]: Checked and it's not needed indeed, removed it: https://github.com/aragon/app/pull/1305/commits/30641aa2d432f9228cc503895eebce4719d653e2

**Review by cgero-eth — APPROVED** [review]
LGTM, thanks @thekidnamedkd!

**Pull request merged**